planner/core/rule_predicate_push_down.go

```diff
@@ -56,9 +56,24 @@ func (p *baseLogicalPlan) PredicatePushDown(predicates []expression.Expression)
 	return nil, p.self
 }
 
+func splitSetGetVarFunc(filters []expression.Expression) ([]expression.Expression, []expression.Expression) {
+	canBePushDown := make([]expression.Expression, 0, len(filters))
+	canNotBePushDown := make([]expression.Expression, 0, len(filters))
+	for _, expr := range filters {
+		if expression.HasUnVectorizableFunc(expr) {
```

Comment:
What if we add more functions into `HasUnVectorizableFunc` later? how about wrapping up the current code of `HasUnVectorizableFunc` into a new function `HasGetSetVarFunc` and call it in `HasUnVectorizableFunc`?